### ISSUE TITLE
How can I train the model without the mask branch?

### ISSUE THREAD

**Issue by ParanoidW** [matterport/Mask_RCNN#141]
I want to train the model with only the bbox input, since my own dataset doesn't have segmentation labels.What should I do with the source code? (I have simply deleted the layers about segmentation, but then I was told that the pretrained weights can't be loaded) @waleedka ,Thanks~

**Comment by waleedka** [comment]
Keras complains if the loaded file has layers that don't exist in the model. For that reason, I added a custom `load_weights()` function that allow you to pass an `exclude` parameter to tell it which weights to skip.

See this thread as well https://github.com/matterport/Mask_RCNN/issues/110

**Comment by ParanoidW** [comment]
Than you so much! 
I just follow the method in #110 and it works. I'm looking forward the single bbox detection version release.

**Comment by dongdongrj** [comment]
@ParanoidW
Hi, have you found the single bbox detection version release? And work ok?

**Comment by ParanoidW** [comment]
@dongdongrj NO.I just follow the method in #110

**Comment by Cyndishine** [comment]
@ParanoidW   
Hi! Can you please tell me how to make your own dataset like coco format?
